experimental/packages/otlp-proto-exporter-base/tsconfig.esnext.json

```diff
@@ -1,21 +1,22 @@
 {
   "extends": "../../../tsconfig.base.esnext.json",
   "compilerOptions": {
-    "allowJs": true,
-    "outDir": "build/esnext",
     "rootDir": "src",
+    "outDir": "build/esnext",
     "tsBuildInfoFile": "build/esnext/tsconfig.esnext.tsbuildinfo"
   },
   "include": [
-    "src/**/*.ts",
-    "src/generated/*.js",
```

Comment:
are you sure this should be removed? We're generating js files from the protobuf definitions. `src/generated/root.js` is not matched by `src/**/*.ts`

Reply:
I just committed the files generated locally during bootstrap without looking deeper int this.

By the way, why do we commit these files if they are generated anyway?

Reply:
They were not generated previously

Reply:
@legendecas I believe changed it so that they are generated. Is this a bug in the generation?

Reply:
I forgot he is out until the end of the month for lunar new year. i'll do more digging